--- mongo-cxx-driver.spec.orig	2024-10-02 16:04:01.333784649 -0400
+++ mongo-cxx-driver.spec	2024-12-02 10:57:02.798895580 -0500
@@ -1,8 +1,10 @@
 # for better compatibility with SCL spec file
 %global pkg_name mongo-cxx-driver
 
+%define _unpackaged_files_terminate_build 0
+
 Name:           mongo-cxx-driver
-Version:        3.6.5
+Version:        4.1.0
 Release:        1%{?dist}
 Summary:        A C++ Driver for MongoDB
 License:        ASL 2.0
@@ -10,9 +12,8 @@
 Source0:        https://github.com/mongodb/%{pkg_name}/archive/%{name}-r%{version}.tar.gz
 
 
-Patch2:         mongo-cxx-driver-3.3.1_paths.patch
-Patch3:         mongo-cxx-driver-catch-update.patch
-BuildRequires:  boost-devel >= 1.49
+BuildRequires:  pkgconfig
+BuildRequires:  pkgconfig(libutf8proc)
 BuildRequires:  openssl-devel
 BuildRequires:  cmake
 BuildRequires:  cyrus-sasl-devel
@@ -67,8 +68,6 @@
 %prep
 %setup -q -n %{name}-r%{version}
 
-%patch2 -p1 -b .paths
-%patch3 -p1 -b .catchupdate
 
 
 %build
@@ -78,7 +77,8 @@
 
 %cmake \
     -DCMAKE_BUILD_TYPE=Release \
-    -DBSONCXX_POLY_USE_BOOST=1 \
+    -DBUILD_VERSION=4.1.0 \
+    -DENABLE_UNINSTALL=OFF \
     .
 
 %cmake_build
