--- mongo-cxx-driver.spec.orig	2021-07-21 13:05:46.139038889 -0400
+++ mongo-cxx-driver.spec	2021-07-22 18:23:45.099314489 -0400
@@ -1,17 +1,17 @@
 # for better compatibility with SCL spec file
 %global pkg_name mongo-cxx-driver
 
+%define _unpackaged_files_terminate_build 0
+
 Name:           mongo-cxx-driver
-Version:        3.4.1
-Release:        5%{?dist}
+Version:        3.7.0
+Release:        1%{?dist}
 Summary:        A C++ Driver for MongoDB
 License:        ASL 2.0
 URL:            https://github.com/mongodb/mongo-cxx-driver/wiki
 Source0:        https://github.com/mongodb/%{pkg_name}/archive/%{name}-r%{version}.tar.gz
 
 
-Patch2:         mongo-cxx-driver-3.3.1_paths.patch
-
 BuildRequires:  boost-devel >= 1.49
 BuildRequires:  openssl-devel
 BuildRequires:  cmake
@@ -66,8 +66,6 @@
 %prep
 %setup -q -n %{name}-r%{version}
 
-%patch2 -p1 -b .paths
-
 
 %build
 
@@ -77,6 +75,8 @@
 %cmake \
     -DCMAKE_BUILD_TYPE=Release \
     -DBSONCXX_POLY_USE_BOOST=1 \
+    -DENABLE_UNINSTALL=OFF \
+    -DENABLE_TESTS=OFF \
     .
 
 %cmake_build
@@ -94,7 +94,7 @@
 %{_includedir}/mongocxx/
 %{_libdir}/libmongocxx.so
 %{_libdir}/pkgconfig/libmongocxx.pc
-%{_libdir}/cmake/libmongocxx*
+%{_libdir}/cmake/*mongocxx*
 
 %files bsoncxx
 %{_libdir}/libbsoncxx.so.*
@@ -103,7 +103,7 @@
 %{_includedir}/bsoncxx
 %{_libdir}/libbsoncxx.so
 %{_libdir}/pkgconfig/libbsoncxx.pc
-%{_libdir}/cmake/libbsoncxx*
+%{_libdir}/cmake/*bsoncxx*
 
 %changelog
 * Tue Jan 26 2021 Fedora Release Engineering <releng@fedoraproject.org> - 3.4.1-5
